backend/api/v1/v1_users/urls.py

```diff
@@ -12,9 +12,10 @@
     re_path(r'^(?P<version>(v1))/login', login),
 
     re_path(r'^(?P<version>(v1))/users', list_users),
-    re_path(r'^(?P<version>(v1))/user/(?P<user_id>[0-9]+)', edit_user),
-    re_path(r'^(?P<version>(v1))/delete/user/(?P<user_id>[0-9]+)',
-            delete_user),
+    re_path(r'^(?P<version>(v1))/user/(?P<user_id>[0-9]+)',
```

Comment:
why do we have two endpoints? we can remove one of them isn't

Reply:
Yes, Just updated